src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -330,10 +330,38 @@
                 <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
                 <target>Este Código de Identificación Bancaria (BIC) no está asociado con el IBAN {{ iban }}.</target>
             </trans-unit>
+            <trans-unit id="86">
+                <source>This value should be valid JSON.</source>
+                <target>Este valor tiene que ser un JSON válido.</target>
+            </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Esta colección tiene que tener exclusivamente elementos únicos.</target>
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Este valor tiene que ser positivo.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Este valor tiene ser positivo o igual a cero.</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Este valor tiene que ser negativo.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Este valor tiene ser negativo o igual a cero.</target>
+            </trans-unit>
             <trans-unit id="92">
                 <source>This value is not a valid timezone.</source>
                 <target>Este valor no es una zona horaria válida.</target>
             </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Esta contraseña ha sido desvelada por un fallo de seguridad y no debería volver a ser utilizada. Por favor utilice otra contraseña</target>
```

Comment:
This message is very difficult to translate. I'd recommend an alternative translation to clearly explain that this is not a security problem of our own application:

```
Esta contraseña no se puede utilizar porque está incluida en un listado de contraseñas públicas obtenido gracias a fallos de seguridad de otros sitios y aplicaciones. Por favor utilice otra contraseña.
```

Reply:
May french translation should be rewieved beacause I've translated it from [this line](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf#L363)

Reply:
The french translation is right btw.

Reply:
@Simperfit I'm not sure about that. First, "data breach" should be translated as [violation de données](https://fr.wikipedia.org/wiki/Violation_de_donn%C3%A9es) instead of "faille de sécurité". Second, the way it's written could mean that the security problem is caused by the web site or application displaying the message.

Reply:
I understand what you mean, for me it does not mean that the current application showing the message is one who has been hacked / selled.
But It can be improved with what you proposed to have the same meaning in french. 

smth like: "Ce mot de passe ne peut pas être utilisé car il fait partie d'une liste de mots de passe publics obtenus grâce aux fuites de données liée à d'autres sites et applications. Veuillez utiliser un autre mot de passe."
WDYT ?

Reply:
It's super long ... but it describes the problem with absolute precision and I think most users can understand it. I wish we could find a more concise message while still explaining things well, but I think we can't.